lib/src/platform.dart

```diff
@@ -127,7 +127,10 @@ DartPlatform classifyPkgPlatform(
   final primaryLibrary =
       _selectPrimaryLibrary(pubspec, transitiveLibs.keys.toSet());
 
-  if (pubspec.hasFlutterKey || pubspec.hasFlutterPluginKey) {
```

Comment:
Wondering if all of these should just be rolled up into one property on pubspec?

Reply:
yeah, I would do that later, after reviewing the very similar Pubspec class in the dart-site project, and unifying all.